dynamic/equal.go

```diff
@@ -179,10 +179,3 @@ func MessagesEqual(a, b proto.Message) bool {
 		return Equal(da, db)
 	}
 }
-
-func MessageName(msg proto.Message) string {
```

Comment:
I've since added a `XXX_MessageName` method to `dynamic.Message`, so it works correctly with `proto.MessageName`. So no need for this alternate method.